presto-hudi/src/main/java/com/facebook/presto/hudi/HudiModule.java

```diff
@@ -105,7 +108,9 @@ protected void setup(Binder binder)
         configBinder(binder).bindConfig(MetastoreClientConfig.class);
         configBinder(binder).bindConfig(ThriftHiveMetastoreConfig.class);
         configBinder(binder).bindConfig(HiveGcsConfig.class);
+        configBinder(binder).bindConfig(HiveAzureConfig.class);
         binder.bind(GcsConfigurationInitializer.class).to(HiveGcsConfigurationInitializer.class).in(Scopes.SINGLETON);
+        binder.bind(AzureConfigurationInitializer.class).to(HiveAzureConfigurationInitializer.class).in(Scopes.SINGLETON);
```

Comment:
Same here, as we have already created `HiveAzureModule`, please re-use that. You can bind it in `HudiConnectorFactory`